clar2wasm/tests/wasm-generation/bitwise.rs

```diff
@@ -0,0 +1,76 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use proptest::proptest;
+
+use crate::{int, uint};
+
+const ONE_OP: [&str; 1] = ["bit-not"];
+const TWO_OPS: [&str; 2] = ["bit-shift-left", "bit-shift-right"];
+const MULTI_OPS: [&str; 3] = ["bit-and", "bit-or", "bit-xor"];
+
+proptest! {
+  #[test]
+  fn crossprop_bitwise_one_op_int(val in int()) {
+    for op in &ONE_OP {
+        crosscheck_compare_only(
+            &format!("({op} {val})")
+        )
+    }
+  }
+}
+
+proptest! {
+  #[test]
+  fn crossprop_bitwise_one_op_uint(val in uint()) {
+    for op in &ONE_OP {
+        crosscheck_compare_only(
+            &format!("({op} {val})")
+        )
+    }
+  }
+}
+
+proptest! {
+  #[test]
+  fn crossprop_bitwise_two_ops_int(val1 in int(), val2 in int()) {
```

Comment:
`val2` needs to be a `uint` for both of the `TWO_OPS` expressions, so these will all fail in both runtimes.

Reply:
Fixed! Thanks for catching that.